config.sub

```diff
@@ -0,0 +1,1860 @@
+#! /bin/sh
+# Configuration validation subroutine script.
+#   Copyright 1992-2021 Free Software Foundation, Inc.
+
+timestamp='2021-03-10'
```

Comment:
also here, fixed timestamp?

Reply:
what is it used for?
these are scripts are copied verbatim from the FSF releases..so wouldn't change them..

Reply:
ok